R/fit_data.R

```diff
@@ -16,13 +16,22 @@
 #'  fit by seeking its minimum.
 #'   * `ks`: Kolmogorov-Smirnov statistic.
 #'   * `likelihood`: Negative log-likelihood.
+#' @param jitter_settings Settings for running the fitting multiple times with
+#'   randomly-generated offsets ('jitter') applied to the starting estimates.
+#'   * `n`: How many jitter iterations to run (default of 7); the total number
+#'   of fits is `n + 1` (because the un-jittered start points are also fit).
+#'   * `prop`: The maximum jitter offset, as a proportion of the start
+#'   value (default of 0.5).
+#'   * `seed`: Seed for the random jitter generator (default is unseeded).
```

Comment:
Question: what does it mean for the default to be unseeded? Is it that it chooses a random seed, that is not saved or displayed in any way, or something else?

Reply:
Never mind, I answered it below :)